templates/server.xml.erb

```diff
@@ -25,6 +25,15 @@
                    <%= key %> = <%= "\"#{value}\"" %>
 <%   end -%>
         />
+<% end -%>
+<% if @tomcat_additional_connectors and ! @tomcat_additional_connectors.empty? -%>
+<% @tomcat_additional_connectors.sort.map do |port, attrs| -%>
```

Comment:
map has had stable sort since Ruby 1.9
so i'd get rid of it, as the result it might surprise people

on second look, everywhere else uses it too, so, yeah, nevermind